README.md

```diff
@@ -10,6 +10,10 @@ Pega Platform is a distributed web application for customer engagement, customer
 
 This *ready* Docker image represents one component of a full image you can use to run a Pega node. It is built on top of Tomcat but does not contain the Pega .war file (hence it is *ready* for the .war file - see [pegasystems/pega on DockerHub][pegasystems/pega] for the full image which *includes* the .war file).
 
+##### User access and control for ready image
+
+This *ready* Docker image runs with built in user i.e. pegauser:pegauser (9001:9001). This has been done to provide limited access policy for image. The file system access is controlled by running image with non-root user. Only required file access has been given to pegauser:pegauser. While constructing your image from *pega-ready* consider the user access and control. Provide required files(or directories) access and ownership to the user. 
+
 ## Constructing your image from *pega-ready*
```

Comment:
## Building a deployable Docker image using this *web-ready* image

For clients who need to build their own deployment image, Pega recommends building your Pega image using your own Dockerfile with contents similar to the example below and specifying the .war file from the Pega distribution kit.  You may also specify a database driver as shown in the example.  It is a best practice to build this image on a Linux system to retain proper file permissions.  Replace the source paths with the actual paths to the Pega Infinity software libraries and specify a valid JDBC driver for your target database to bake it in.